styles/globals.css

```diff
@@ -19,8 +23,9 @@ a {
   html {
     color-scheme: dark;
   }
+
   body {
     color: white;
     background: black;
   }
-}
```

Comment:
set your editor to insert newline at end of file. every file should end with newline char